dotnet-client-libraries/YamlDotNet/App/UseCases/ObjectGraphSerializeAndDeserialize.cs

```diff
@@ -0,0 +1,21 @@
+using App.Models;
+using YamlDotNet.Serialization;
+
+namespace App.UseCases;
+
+public static class ObjectGraphSerializeAndDeserialize
+{
+    public static string SerializeStore(Store store)
+    {
+        var serializer = new SerializerBuilder().Build();
+        var yaml = serializer.Serialize(store);
```

Comment:
Looks like you can just return this directly.

```suggestion
        return serializer.Serialize(store);
```

Reply:
@grantwinney Done